tests/test_hyperlinks.py

```diff
@@ -54,3 +54,12 @@ def test_hyperlinks_do_not_contain_prohibited_characters(self):
                 re.search(r'[<>]', h['name']) is None,
                 msg='Hyperlink "%s" contains forbidden characters in "%s".' % (h['md'], h['file'])
             )
+
+    def test_tag_hyperlinks(self):
+        """Test that a tag-like hyperlinks are valid."""
+        for h in self.hyperlinks:
+            if h['name'] in ['C++', 'Java', 'Python', 'ROS', 'Matlab']:
```

Comment:
The correct spelling for MATLAB is all-caps. Currently our doc has a mixture of both... 😒